plugin/pkg/scheduler/scheduler.go

```diff
@@ -46,10 +50,11 @@ type Scheduler struct {
 type Config struct {
 	// It is expected that changes made via SchedulerCache will be observed
 	// by NodeLister and Algorithm.
-	SchedulerCache schedulercache.Cache
-	NodeLister     algorithm.NodeLister
-	Algorithm      algorithm.ScheduleAlgorithm
-	Binder         Binder
+	SchedulerCache      schedulercache.Cache
+	NodeLister          algorithm.NodeLister
+	Algorithm           algorithm.ScheduleAlgorithm
+	Binder              Binder
+	PodConditionUpdater PodConditionUpdater
```

Comment:
We are updating pod condition to true atomically together with binding. In scheduler.go there's a comment about that. Where do you suggest to add this comment?